src/App.tsx

```diff
@@ -22,33 +69,28 @@ export const App: React.FC = () => {
 
         <div className="dropdown-menu" role="menu">
           <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Petronella de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Hercke</p>
-            </div>
+            {preparedPeopleData.length !== 0
+              ? (preparedPeopleData.map(currentPerson => (
+                <a
+                  onClick={(event) => handleClick(event, currentPerson)}
+                  href={`#${currentPerson.slug}`}
+                  key={currentPerson.slug}
+                  className="dropdown-item"
+                >
+                  <p
+                    className={`has-text-${currentPerson.sex === 'f' ? 'danger' : 'link'}`}
+                  >
+                    {currentPerson.name}
+                  </p>
+                </a>
+              )))
+              : (
+                <p
+                  className="subtitle is-6"
+                >
+                  No matching suggestions
+                </p>
+              )}
```

Comment:
looks like code mismatch with deployed version. 
still renders empty dropdown in case when no matching suggestions.
try to deploy again
![Screenshot from 2023-08-16 12-53-48](https://github.com/mate-academy/react_autocomplete/assets/33029864/85efcf91-a670-4b04-b40f-32d980f890e9)